src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -901,32 +910,98 @@ def test_get_job_logs_ok(self):
                 self.assertIn(str(first + idx), line["line"])
                 self.assertEqual(0, line["is_error"])
 
-    @mock.patch(CLIENTS, get_failing_mock_client)
+    @mock.patch(CLIENTS, get_mock_client)
     def test_get_job_logs_failure(self):
-        job_id = JOB_COMPLETED
+        job_id = JOB_CREATED
         req = make_comm_msg(LOGS, job_id, False)
-        with self.assertRaises(NarrativeException) as e:
-            self.jc._handle_comm_message(req)
-        self.assertIn("Can't get job logs", str(e.exception))
+        self.jc._handle_comm_message(req)
         msg = self.jc._comm.last_message
-        self.assertEqual(ERROR, msg["data"]["msg_type"])
-        self.assertEqual("Unable to retrieve job logs", msg["data"]["content"]["error"])
+        print(msg)
+        self.assertEqual(
+            msg["data"],
+            {
+                "msg_type": LOGS,
+                "content": {
+                    JOB_CREATED: {
+                        "job_id": JOB_CREATED,
+                        "batch_id": None,
+                        "error": "Cannot find job log with id: " + JOB_CREATED,
+                    }
+                },
+            },
+        )
 
     def test_get_job_logs_no_job(self):
         job_id = None
         req = make_comm_msg(LOGS, job_id, False)
-        err = JobIDException(JOB_NOT_REG_ERR, job_id)
-        with self.assertRaisesRegex(type(err), str(err)):
+        err = JobIDException(JOBS_MISSING_FALSY_ERR, [job_id])
+        with self.assertRaisesRegex(type(err), re.escape(str(err))):
             self.jc._handle_comm_message(req)
-        self.check_error_message(LOGS, {JOB_ID: job_id}, err)
+        self.check_error_message(LOGS, {JOB_ID_LIST: [job_id]}, err)
 
+    @mock.patch(CLIENTS, get_mock_client)
     def test_get_job_logs_bad_job(self):
         job_id = "bad_job"
         req = make_comm_msg(LOGS, job_id, False)
-        err = JobIDException(JOB_NOT_REG_ERR, job_id)
-        with self.assertRaisesRegex(type(err), str(err)):
-            self.jc._handle_comm_message(req)
-        self.check_error_message(LOGS, {JOB_ID: job_id}, err)
+        self.jc._handle_comm_message(req)
```

Comment:
these behaviours have changed -- a `job_logs` message is not returned with the content being an error message